js/interface.js

```diff
@@ -94,6 +95,43 @@ var DynamicLists = (function() {
     _this.init();
   }
 
+  function initDataSourceProvider(currentDataSourceId) {
+    var dataSourceData = {
+      dataSourceTitle: 'Profile data source',
```

Comment:
The LFD doesn't need to select a **Profile data source**. This seems inappropriately labelled.